docs/upgrade/mongodb_major_upgrades.md

```diff
@@ -0,0 +1,36 @@
+# MongoDB: Major version upgrades
+
+Starting with Percona Everest 1.6.0, you can upgrade your MongoDB database versions, allowing upgrades with minimal downtime and disruption.
+
+## Before you upgrade
+
+The prerequisites for performing a major version upgrade of MongoDB in Percona Everest are:
+
+- **Version compatibility:** Verify that your current MongoDB version is eligible for an upgrade.
+
+- **System resources:** Ensure that your cluster has adequate CPU, memory, and storage to handle the upgrade process.
```

Comment:
How can the user know about this stuff? I don't think that there's an action to be had here, I'd remove this section

Reply:
Done.